wd.sh

```diff
@@ -256,20 +256,45 @@ wd_remove()
 }
 
 wd_browse() {
+    # Check if fzf is installed
     if ! command -v fzf >/dev/null; then
         echo "This functionality requires fzf. Please install fzf first."
         return 1
     fi
+
+    # Ensure wd_config_file is properly set
+    if [[ -z $wd_config_file ]]; then
+        wd_config_file="${WD_CONFIG:-$HOME/.warprc}"
+    fi
+
+    # Check if config file exists
+    if [[ ! -f $wd_config_file ]]; then
+        return 1
+    fi
```

Comment:
Why have you added these two checks? This should be being done every time the script is run: https://github.com/mfaerevaag/wd/blob/0c40cff2ade3e93cda91d09e678c8f9a6430cd04/wd.sh#L448-L461

Reply:
https://github.com/mfaerevaag/wd/commit/13277d2cb1d8bfaae950896bf0f44fb46c729f00 has removed the global variables for the config file. If we want to do it this way, we need to make some changes. Please review: